src/algebra/group_power/lemmas.lean

```diff
@@ -185,6 +185,36 @@ by rw [zpow_add, zpow_one]
 theorem zpow_bit1 (a : G) (n : ℤ) : a ^ bit1 n = a ^ n * a ^ n * a :=
 by rw [bit1, zpow_add, zpow_bit0, zpow_one]
 
+/-- To show a property of all powers of `g` it suffices to show it is closed under multiplication
+by `g` and `g⁻¹` on the left. -/
+@[to_additive "To show a property of all multiples of `g` it suffices to show it is closed under
+addition by `g` and `-g` on the left."]
+lemma zpow_induction_left {g : G} {P : G → Prop} (h_one : P (1 : G))
```

Comment:
This is pretty much just a special (and slightly more convenient) case of `subgroup.closure_induction_left`, right? Can you include a reference to it and the analogues in the four docstrings? I'm ok with having a special case in this early file as long as it admits to a more general form existing elsewhere.